tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/ClientCredentialsTests.WithRegion.cs

```diff
@@ -68,22 +68,21 @@ public async Task AcquireTokenToRegionalEndpointAsync()
         }
 
         [TestMethod]
-        [Ignore] //See https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2781
-        public async Task InvalidRegion_GoesToInvalidAuthority_Async()
+        public async Task RequestGoesToUserSpecifiedRegion_Async()
         {
             // Arrange
             var factory = new HttpSnifferClientFactory();
             var settings = ConfidentialAppSettings.GetSettings(Cloud.Public);
-            _confidentialClientApplication = BuildCCA(settings, factory, true, "invalid");
+            _confidentialClientApplication = BuildCCA(settings, factory, true, "westus");
```

Comment:
The point of this test is to make sure that MSAL & ESTS-R behave consistently when a bad region is used. MSAL doesn't know what a "bad" region is, if the developer tells us to use "XZY", we'll use "XZY". I expect ESTS-R to be able to serve this request.

As such: 

- the original name of the test still makes sense to me
- `westus` is not a valid region name, so you are testing the right thing :)
- If you use `invalid` what happens? `invalid` is more descriptive.

Reply:
When using invalid the following error is produced: The remote name could not be resolved: 'invalid.login.microsoft.com

Perhaps ESTS-R changed the behavior of this specific address?

Reply:
@bgavrilMS 
- Yes, initially `InvalidRegion_GoesToInvalidAuthority_Async` tested the behavior you describe and we did expect it to work with ESTS-R. But as Travis tested, invalid regions don't work.
- `westus` is a valid region, that's why the test passes with it.
- Since invalid regions don't work, I suggested to rename the test to `RequestGoesToUserSpecifiedRegion_Async` and test that user-specified region takes precedence over the auto detected one.

Reply:
so did the behavior from ests-r change or did we not test correctly? If the behavior changed, we should tell them, as they just broke our customers. Based on what i'm understanding, the customer can give any region and that host should be used, and if it's not available, ests-r should serve the request from the global endpoint. But that's no happening, we are getting an error. is that correct? and before it was working with an invalid region?